xml/templates/message_templates/payment_or_refund_notification_html.tpl

```diff
@@ -47,19 +59,19 @@
   </tr>
   <tr>
     <td {$labelStyle}>
-      {ts}You Paid{/ts}
+      {ts}Totally Paid{/ts}
```

Comment:
Total of payments made?

Reply:
The Total amount seems to reflect the net total amount paid including refunds and payments. I will change the tpl to make it easier to comprehend.

Reply:
Hi @eileenmcnaughton I restructured the table that is shown in the payment & refund email. I included screenshots. I think they are much clearer for the end user to understand now. Also the payment & refund details are more alike. And finally I think this setup already anticipates the info needed in the emails to reflect partial refunds.

Reply:
@magnolia61 - just - it's just that 'Totally Paid' doesn't read well. Total Paid would work better perhaps.

Reply:
Total Paid would be perfectly fine. It could have been changed in https://github.com/civicrm/civicrm-core/pull/15749 but can also change later probably.